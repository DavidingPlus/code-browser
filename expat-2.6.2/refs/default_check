<def f='expat-2.6.2/tests/handlers.h' l='574' ll='578'/>
<use f='expat-2.6.2/tests/handlers.h' l='574'/>
<size>16</size>
<mbr r='default_check::expected' o='0' t='const XML_Char *'/>
<mbr r='default_check::expectedLen' o='64' t='const int'/>
<mbr r='default_check::seen' o='96' t='XML_Bool'/>
