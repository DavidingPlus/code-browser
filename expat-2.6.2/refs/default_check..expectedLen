<dec f='expat-2.6.2/tests/handlers.h' l='576' type='const int'/>
<offset>64</offset>
<use f='expat-2.6.2/tests/handlers.c' l='1919' u='r' c='checking_default_handler'/>
