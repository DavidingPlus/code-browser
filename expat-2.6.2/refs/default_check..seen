<dec f='expat-2.6.2/tests/handlers.h' l='577' type='XML_Bool'/>
<offset>96</offset>
<use f='expat-2.6.2/tests/basic_tests.c' l='5175' u='r' c='test_default_doctype_handler'/>
<use f='expat-2.6.2/tests/handlers.c' l='1921' u='w' c='checking_default_handler'/>
