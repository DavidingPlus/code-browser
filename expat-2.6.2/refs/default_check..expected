<dec f='expat-2.6.2/tests/handlers.h' l='575' type='const XML_Char *'/>
<offset>0</offset>
<use f='expat-2.6.2/tests/basic_tests.c' l='5174' u='r' c='test_default_doctype_handler'/>
<use f='expat-2.6.2/tests/handlers.c' l='1918' u='r' c='checking_default_handler'/>
<use f='expat-2.6.2/tests/handlers.c' l='1920' u='r' c='checking_default_handler'/>
